### DISCUSSION TITLE
Solving Large Scale Optimal Transport Problem

### DISCUSSION THREAD

**Discussion by akul-goyal** [cvxpy/cvxpy#1742] [Q&A]
Hi,

I am interested in solving a large-scale optimal transport problem (10,000 source nodes, 20,000 destination nodes) and I am trying to find the best solver to use. Given it is an optimal transport problem, I am using highly sparse matrics and I wonder which solver is able to take the most advantage of this to provide a solution within a reasonable time.

**Comment by SteveDiamond** [comment]
Basically all the solvers exploit sparsity. The best solver depends on what licenses you can use (GPL, commercial). I would suggest exploring the list here: https://www.cvxpy.org/tutorial/advanced/index.html#choosing-a-solver
